libs/form-builder/src/components/formFooter.tsx

```diff
@@ -0,0 +1,11 @@
+import { Button } from '@daohaus/ui';
+import React from 'react';
+
+export const FormFooter = ({ submitDisabled }: { submitDisabled: boolean }) => {
+  /*Form Alert Component goes here*/
```

Comment:
What is the Form Alert component? I'm sure that this is something I should already know 😅

Reply:
It's that alert that goes near the submit button. It lets you know that you're not on the right network or whatever.